src/css/librarybrowser.css

```diff
@@ -421,9 +422,15 @@
     left: .8em
 }
 
+.noBackdrop {
+    display: none;
+}
+
```

Comment:
Didn't you remove this for now?

Reply:
for now i removed it but the reason i used that class  b/c of this on person detail
![Annotation 2020-01-05 234756](https://user-images.githubusercontent.com/32230989/71785895-09cc0200-3016-11ea-8e27-2c5c34a975cc.jpg)